modules/web/src/core/editor/constants.js

```diff
@@ -4,6 +4,7 @@ export const COMMANDS = {
     UNDO: 'undo',
     REDO: 'redo',
     ACTIVATE_EDITOR_FOR_FILE: 'activate-editor-for-file',
+    SHOW_LEFT_PANEL: 'show-left-panel',
```

Comment:
this is not needed right, it's defined in layout events (layout-plugin/constant.js)